ow_lander/urdf/lander.xacro

```diff
@@ -543,6 +543,13 @@
       </plugin>
     </visual>
     <selfCollide>1</selfCollide>
+    <collision>

```

Comment:
You could just leave this block out to have collisions with terrain. It would probably be good to at least comment that this is enabling collisions with terrain.

Reply:
I figured it would be much easier this rather than looking for the block in code history to find how to disable or enable collision with the terrain.